src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -127,112 +134,121 @@
                                                                       receiver-network-values))
-            suggested-routes                          (rf/sub [:wallet/wallet-send-suggested-routes])
-            routes                                    (when suggested-routes
-                                                        (or (:best suggested-routes) []))
-            no-routes-found?                          (and
-                                                       (every-network-value-is-zero?
-                                                        sender-network-values)
-                                                       (not (nil? routes))
-                                                       (not loading-routes?)
-                                                       (not token-not-supported-in-receiver-networks?))
-            receiver-networks                         (rf/sub [:wallet/wallet-send-receiver-networks])
-            receiver-preferred-networks               (rf/sub
-                                                       [:wallet/wallet-send-receiver-preferred-networks])
-            receiver-preferred-networks-set           (set receiver-preferred-networks)
-            sending-to-unpreferred-networks?          (not (every? (fn [receiver-selected-network]
-                                                                     (contains?
-                                                                      receiver-preferred-networks-set
-                                                                      receiver-selected-network))
-                                                                   receiver-networks))]
+            suggested-routes (rf/sub [:wallet/wallet-send-suggested-routes])
+            routes (when suggested-routes
+                     (or (:best suggested-routes) []))
+            no-routes-found? (and
+                              (every-network-value-is-zero?
+                               sender-network-values)
+                              (not (nil? routes))
+                              (not loading-routes?)
+                              (not token-not-supported-in-receiver-networks?))
+            receiver-networks (rf/sub [:wallet/wallet-send-receiver-networks])
+            receiver-preferred-networks (rf/sub
+                                         [:wallet/wallet-send-receiver-preferred-networks])
+            receiver-preferred-networks-set (set receiver-preferred-networks)
+            sending-to-unpreferred-networks? (not (every? (fn [receiver-selected-network]
+                                                            (contains?
+                                                             receiver-preferred-networks-set
+                                                             receiver-selected-network))
+                                                          receiver-networks))
+            input-error (controlled-input/input-error input-state)]
```

Comment:
@ulisesmac I mean this long list of definitions which is a mix of event, subs, utils, and values, could introduce bugs in the future and could be harder to debug. We need to have more separation of concerns here